shelley/design-spec/delegation_design_spec.tex

```diff
@@ -461,31 +460,30 @@ \subsection{Overview of Delegation}
     %% Nodes
     %%
     \node (owner)   [concept]                    {Owner};
-    \node (addr)    [entity, left = of owner]    {Pubkey address};
-    \node (skey)    [entity, below = of owner]   {Staking key};
-    \node (pkey)    [entity, left = of skey]     {Payment key};
-    \node (racc)    [entity, right = of skey]    {Reward account /
-      Reward address};
+    \node (valaddr)    [entity, left = of owner]    {Value address};
+    \node (registeredStakeAddr)    [entity, below = of owner]   {Registered stake address};
+    \node (pkey)    [entity, left = of registeredStakeAddr]     {Payment key (hash)\\or script (hash)};
```

Comment:
I think we decided on `payment object` and `staking object` which comprises key or some script

Reply:
So should I replace `Payment key (hash)\\or script (hash)` by `Payment object`? BTW, payment object is not defined ...

Reply:
true, maybe leave it for now as `key or script`, we can use object in the formal spec